sympy/combinatorics/permutations.py

```diff
@@ -3020,3 +3012,10 @@ def __new__(cls, perm, x, evaluate=None):
 
         obj = super().__new__(cls, perm, x)
         return obj
+
+
+@dispatch(Permutation, Permutation)
+def _eval_Eq(lhs, rhs):
+    if lhs._size != rhs._size:
+        return None
+    return as_Boolean(lhs._array_form == rhs._array_form)
```

Comment:
I wonder if `_eval_Eq` should return a fuzzy bool rather than a `Boolean`...

Reply:
that makes sense to me